src/main/java/frc/robot/subsystems/funnel/commands/StartFunnel.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.subsystems.funnel.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.funnel.Funnel;
+
+/**
+ * this command activates Funnel's motor
+ */
+public class StartFunnel extends CommandBase {
+
+    private Funnel funnel;
+    private boolean direction;

```

Comment:
please give this variable a better name, like `isMovingUp` or something similar to help understanding which direction is the `true`